client/components/App.jsx

```diff
@@ -30,16 +31,27 @@ class App extends React.Component {
       });
   }
 
+  comparisonToggle(relatedProduct, relatedStyle) {
+    // console.log(this.state.currentProduct);
+    console.log(relatedStyle);
+    var status = !this.state.comparisonToggle ? <Comparison_Model toggleComparison={this.comparisonToggle} displayedProduct={this.state.currentProduct} relatedProduct={relatedProduct} /> : false;
 
+    this.setState({
+      comparisonToggle: status
+    })
+  }
 
   render() {
+    var comparison = this.state.comparisonToggle ? this.state.comparisonToggle : <div></div>;
```

Comment:
this is cool! never thought about using render like this

Reply:
Yeah its pretty useful!